microsoft/testsuites/performance/nvmeperf.py

```diff
@@ -72,3 +73,52 @@ def perf_nvme(self, node: Node, result: TestResult) -> None:
             disk_type=DiskType.nvme,
             test_result=result,
         )
+
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test NVMe disk performance
+        with 'io_uring' as ioengine.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            supported_features=[Nvme],
+        ),
+    )
+    def perf_nvme_io_uring(self, node: Node, result: TestResult) -> None:
+        nvme = node.features[Nvme]
```

Comment:
use a method for common code used in perf_nvme_io_uring and perf_nvme

Reply:
Added common method "_perf_nvme" as suggested